ivy/functional/frontends/jax/config.py

```diff
@@ -1,7 +1,7 @@
-jax_enable_x64 = False
```

Comment:
`func_wrapper` files would also benefit from an alphabetic re-ordering. I guess you have excluded them but could you have them be sorted in 2 parts, first helpers and then the actual functions? I think that would be great.